src/ucp/core/ucp_context.c

```diff
@@ -295,6 +295,14 @@ static ucs_config_field_t ucp_config_table[] = {
    "Experimental: enable new protocol selection logic",
    ucs_offsetof(ucp_config_t, ctx.proto_enable), UCS_CONFIG_TYPE_BOOL},
 
+  {"KEEPALIVE_TIMEOUT", "0us",
```

Comment:
should the default be auto?

Reply:
auto? what is criteria?

Reply:
some default value, otherwise keepalive will remain disabled even if user requests it

Reply:
yep, will update it when more transports are supported to not break existing tests